wrapper.html

```diff
@@ -284,7 +283,7 @@ <h3 class="admin-menu-title text-small text-caps text-font-secondary margin-bott
 	</div>
 </footer>
 
-<script src="https://dbqvwi2zcv14h.cloudfront.net/ak/ak-v3.js"></script>
+<script src="https://dbqvwi2zcv14h.cloudfront.net/ak-dev/ak-v3.js"></script>

```

Comment:
@mrkimani-ndegwa same note as my comment above.

Reply:
Good point. Is there a difference between what ak and ak-dev has?

Reply:
I'm not sure if the `ak` folder has some of the new assets that were added to `ak-dev`, eg. the monthly-heart icon. We'll just need to add those new assets to `ak` if they're not there already. I'll take a look and do that if needed.

Reply:
@kimani-350 just letting you know that the only missing file in the `ak` key (just learned that S3 refers to these 'folders' as keys) was monthly-heart.svg, so I just copied that into `ak`.